src/components/browse/components/FacetList/FacetTermsList.js

```diff
@@ -284,18 +322,27 @@ export class FacetTermsList extends React.PureComponent {
                     </div>
                     { indicator }
                 </h5>
-                <ListOfTerms {...{ facet, facetOpen, terms, persistentCount, onTermClick, expanded, getTermStatus, termTransformFxn }} onToggleExpanded={this.handleExpandListToggleClick} />
+                <ListOfTerms {...{ facet, facetOpen, terms, onTermClick, expanded, getTermStatus, termTransformFxn, searchText, schemas, persistentCount, defaultBasicSearchAutoDisplayThreshold }} onSaytTermSearch={this.handleSaytTermSearch} onBasicTermSearch={this.handleBasicTermSearch} onToggleExpanded={this.handleExpandListToggleClick} />
             </div>
         );
     }
 }
 FacetTermsList.defaultProps = {
-    'persistentCount' : 10
+    'persistentCount' : 10,
+    'defaultBasicSearchAutoDisplayThreshold': 15
 };
 
-
 const ListOfTerms = React.memo(function ListOfTerms(props){
-    const { facet, facetOpen, terms, persistentCount, onTermClick, expanded, onToggleExpanded, getTermStatus, termTransformFxn } = props;
+    const { facet, facetOpen, terms, onTermClick, expanded, onToggleExpanded, getTermStatus, termTransformFxn, searchText, onBasicTermSearch, onSaytTermSearch, persistentCount, defaultBasicSearchAutoDisplayThreshold } = props;
+    let { search_type: searchType = 'none' } = facet;
```

Comment:
(Low priority / doesn't matter too much) We could change this from `'none'` to `null`, benefit being that comparing null === null takes faster than 'none' === 'none' in the line below, but drawback being the change in type. I guess this subjective to large extent depending on how much prefer static types vs think the dynamic typing of JS is a benefit lol. I'm personally bit more in the latter camp, mostly because to maintain static types a good practice in my opinion might be to use something like TypeScript, but don't really want to take it on given other more-pressing priorities.